custom_components/tuya_local/devices/catit_pixi_smart_feeder.yaml

```diff
@@ -22,18 +27,25 @@ entities:
     category: config
     hidden: true
     dps:
+      # Not thought to be used with Catit 43752
       - id: 1
         type: base64
         name: value
         optional: true
   - entity: sensor
     icon: "mdi:paw"
-    name: Feed report
+    name: Last meal size
```

Comment:
Renaming entities after release causes problems for existing users, who may have automations referencing this.